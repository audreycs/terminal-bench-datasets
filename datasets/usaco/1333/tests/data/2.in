10
85 13
71 66
28 66
29 71
96 63
38 31
17 68
36 58
36 4
63 33
