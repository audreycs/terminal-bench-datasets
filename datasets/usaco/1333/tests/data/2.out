20
32
12
15
27
16
36
12
15
17
